site/src/pages/CreateTemplatePage/CreateTemplateForm.tsx

```diff
@@ -608,16 +613,20 @@ export const CreateTemplateForm: FC<CreateTemplateFormProps> = (props) => {
                     <strong>Allow everyone to use the template</strong>
 
                     <HelpTooltip>
-                      <HelpTooltipText>
-                        If unchecked, only users with the &apos;template
-                        admin&apos; and &apos;owner&apos; role can use this
-                        template until the permissions are updated. Navigate to{" "}
-                        <strong>
-                          Templates &gt; Select a template &gt; Settings &gt;
-                          Permissions
-                        </strong>{" "}
-                        to update permissions.
-                      </HelpTooltipText>
+                      <HelpTooltipTrigger size="small" />
+                      <HelpTooltipContent>
+                        <HelpTooltipText>
+                          If unchecked, only users with the &apos;template
+                          admin&apos; and &apos;owner&apos; role can use this
+                          template until the permissions are updated. Navigate
+                          to{" "}
+                          <strong>
+                            Templates &gt; Select a template &gt; Settings &gt;
```

Comment:
I know it was like this before, but &gt; is not good here, especially for screen readers. should be `&rarr;` or something

Reply:
Why it would be better than `&gt;`?

Reply:
because screen readers will read this as "Templates greater than select a template greater than settings greater than" which is nonsense and inaccessible

Reply:
Ah I see, I think I know what we can do here and keep the esthetic of the ">"